### PR TITLE
Updated the Upcoming Events list UI on the Organization dashboard.

### LINKED ISSUE #1358: On Dashboard page upcoming event list in not aligned correctly.
**Describe the bug**
On the Dashboard page, the upcoming event list is not aligned correctly.

**To Reproduce**
Steps to reproduce the behaviour:

1. Create an Organization.
2. Now, go to the Dashboard page.
3. Now, see the bug in the upcoming Events list.

**Expected behaviour**
The events and the location details must be aligned correctly, irrespective of the length of the event title.

**Actual behaviour**
On the Dashboard page, the upcoming event list should be aligned correctly.

**Screenshots**
image

**Additional details**
I Would like to work on this issue :)

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AmitSharma512**
**What kind of change does this PR introduce?**

Updated the Upcoming Events list UI on the Organization dashboard.

**Issue Number:**

Fixes #1358 

**Snapshots/Videos:**

image

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: src/components/OrganizationDashCards/CardItem.module.css [hunk 1]
```diff
@@ -2,7 +2,13 @@
   position: relative;
   display: flex;
   align-items: center;
-  padding: 0.75rem 0;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
```
Comment (rishav-jha-mech) [inline src/components/OrganizationDashCards/CardItem.module.css:6]: change this to a single line
```css
padding: 1rem 3rem 1rem 1.5rem;
```

**Comment by rishav-jha-mech** [comment]
@AmitSharma512 you need to import the icon, not the circle or background associated with them, otherwise we cannot use the same icon else where, also don't add "new" to the file name of icons. Use flex inside the time element, so that instead of overflowing it moves to the next line, for Date its okay but if there is time involved too, it may overflow

File: src/components/OrganizationDashCards/CardItem.module.css [hunk 2]
```diff
@@ -2,7 +2,10 @@
   position: relative;
   display: flex;
   align-items: center;
-  padding: 0.75rem 0;
+  padding: 1rem 3rem 1rem 1.5rem;
+  border: 1px solid #dddddd;
```
Comment (rishav-jha-mech) [inline src/components/OrganizationDashCards/CardItem.module.css:6]: use css variable here

File: src/components/OrganizationDashCards/CardItem.module.css [hunk 3]
```diff
@@ -39,11 +42,36 @@
 .cardItem .title {
   font-size: 1rem;
   flex: 1;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 18px;
+  color: #000000;
```
Comment (rishav-jha-mech) [inline src/components/OrganizationDashCards/CardItem.module.css:48]: Use css variable from bootstrap here too

**Comment by rishav-jha-mech** [comment]
@AmitSharma512 datesnew.svg and eventsnew.svg are the same use any one only and i see no use of appending new to their file name so please remove that, I can see you have not applied inherit to the colour props in them, the svgs should not have any colour by default, so that we can reuse wherever we want in our project. Please follow how we are using svgs in Leftdrawers, you will get the idea

File: src/components/OrganizationDashCards/CardItem.module.css (hunk 2 above)
Comment (AmitSharma512) [reply src/components/OrganizationDashCards/CardItem.module.css:6]: @rishav-jha-mech can you give me some idea about how to use the CSS variables ?

Comment (rishav-jha-mech) [reply src/components/OrganizationDashCards/CardItem.module.css:6]: like this,
```css
border: 1px solid var(--bs-gray-200);
```
`--bs-gray-200` this variable is from bootstrap

**Comment by AmitSharma512** [comment]
> @AmitSharma512 datesnew.svg and eventsnew.svg are the same use any one only and i see no use of appending new to their file name so please remove that, I can see you have not applied inherit to the colour props in them, the svgs should not have any colour by default, so that we can reuse wherever we want in our project. Please follow how we are using svgs in Leftdrawers, you will get the idea

@rishav-jha-mech datesnew ad eventsnew are different, they are the new logo from the figma file that's why i have appended new to their name.

**Comment by AmitSharma512** [comment]
> @AmitSharma512 datesnew.svg and eventsnew.svg are the same use any one only and i see no use of appending new to their file name so please remove that, I can see you have not applied inherit to the colour props in them, the svgs should not have any colour by default, so that we can reuse wherever we want in our project. Please follow how we are using svgs in Leftdrawers, you will get the idea

@rishav-jha-mech I can only download the default color logo from the figma file, i cannot download the SVG without any color as i have only preview access to the figma file.

File: src/components/OrganizationDashCards/CardItem.module.css (hunk 2 above)
Comment (AmitSharma512) [reply src/components/OrganizationDashCards/CardItem.module.css:6]: @rishav-jha-mech, can I write inline bootstrap CSS variables in the classname? Is that ok to write ??

**Comment by rishav-jha-mech** [comment]
@AmitSharma512 you need to modify the svg file by yourself, see for these attributes in svg file like
```svg
fill = "#some-colour-hash"
```
OR
```svg
stroke = "#some-colour-hash"
```

and change it to 
```svg
fill = "current"
or
stroke = "current"
```

Check LeftDrawer component for reference

**Comment by AmitSharma512** [comment]
@rishav-jha-mech please review it

File: src/assets/svgs/locationnew.svg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M7.99999 14.1257L8.48075 13.584C9.02627 12.9593 9.51694 12.3665 9.95351 11.8027L10.3139 11.3273C11.8187 9.29983 12.5714 7.69069 12.5714 6.50136C12.5714 3.96269 10.5249 1.90479 7.99999 1.90479C5.47503 1.90479 3.42856 3.96269 3.42856 6.50136C3.42856 7.69069 4.18132 9.29983 5.68608 11.3273L6.04646 11.8027C6.66927 12.6007 7.32088 13.3751 7.99999 14.1257Z" stroke="#31BB6B" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M7.99998 8.38093C9.05195 8.38093 9.90474 7.52814 9.90474 6.47617C9.90474 5.4242 9.05195 4.57141 7.99998 4.57141C6.94801 4.57141 6.09521 5.4242 6.09521 6.47617C6.09521 7.52814 6.94801 8.38093 7.99998 8.38093Z" stroke="#31BB6B" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (rishav-jha-mech) [inline src/assets/svgs/locationnew.svg:4]: Why there is hardcoded value for stroke, add current there also

**Comment by rishav-jha-mech** [comment]
@AmitSharma512 fix the failing tests

**Comment by AmitSharma512** [comment]
@rishav-jha-mech made the required changes; please review it.

**Review by beingnoble03 — COMMENTED** [review]
@AmitSharma512 please rename the svg files with something relevant to their use. Eg. `cardItemDate`, `cardItemEvent` and `cardItemLocation`.

**Review by beingnoble03 — APPROVED** [review]
Rest LGTM.

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
@rishav-jha-mech Please review. You had some requested changes that need to be verified

**Comment by palisadoes** [comment]
Please fix the introspection error

**Comment by AmitSharma512** [comment]
> Please fix the introspection test error

@palisadoes, it is not because of the files changed by me; the error is in the mutations file, and I haven't changed the mutation files. I am getting this error because of the previous commit I pulled from the branch.

**Comment by AmitSharma512** [comment]
> Please fix the introspection test error

@palisadoes The error is because of the files changed in the PR #1384

**Comment by AmitSharma512** [comment]
> @rishav-jha-mech Please review. You had some requested changes that need to be verified

All the suggestions from @rishav-jha-mech have been made, you can verify them.

**Comment by AmitSharma512** [comment]
@beingnoble03 can you please review the changes requested by @rishav-jha-mech , so that I can work on new issues :)))

**Pull request merged**